core/renderers/block_rendering_rewrite/block_rendering.js

```diff
@@ -34,21 +34,27 @@ goog.require('Blockly.blockRendering.Debug');
 goog.require('Blockly.blockRendering.Drawer');
 goog.require('Blockly.blockRendering.RenderInfo');
 goog.require('Blockly.blockRendering.ConstantProvider');
-goog.require('Blockly.blockRendering.HighlightConstantProvider');
+goog.require('Blockly.geras.HighlightConstantProvider');
 
 goog.require('Blockly.geras.RenderInfo');
+goog.require('Blockly.geras.Drawer');
 goog.require('Blockly.thrasos.RenderInfo');
 goog.require('Blockly.zelos.RenderInfo');
 
+// TODO (#2702): Pick an API for choosing a renderer.
+Blockly.blockRendering.rendererName = 'geras';
```

Comment:
Update: I'm moving forward with the gross version, mostly because it means i don't have to finish defining all the bits and bobs on the different renderers yet.  I want to move to the version you're suggesting, but have a few more changes to make that will make it easier to do.